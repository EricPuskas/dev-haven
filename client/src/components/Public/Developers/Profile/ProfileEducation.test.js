import { shallow } from "enzyme";
import React from "react";
import ProfileEducation from "./ProfileEducation";

it("expect to render Profile Education component", () => {
  const mockProps = {
    education: [
      {
        school: "Lorem Ipsum",
        degree: "Lorem Ipsum",
        fieldofstudy: "Lorem Ipsum",
        from: "test",
        to: "test",
        description: ""
      }
    ]
  };
  expect(
    shallow(<ProfileEducation {...mockProps} />).debug()
  ).toMatchSnapshot();
});
