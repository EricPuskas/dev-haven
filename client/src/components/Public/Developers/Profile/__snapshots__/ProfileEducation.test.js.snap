// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Profile Education component 1`] = `
"<div>
  <h3 className=\\"text-dark\\" />
  <p>
    <t format=\\"YYYY/MM/DD\\" element={{...}} fromNow={false} toNow={false} calendar={false} ago={false} unix={false} utc={false} local={false} unit={{...}} withTitle={false} decimal={false} titleFormat=\\"\\" interval={60000} filter={[Function: filter]} onChange={[Function: onChange]} />
     -
     
    <t format=\\"YYYY/MM/DD\\" element={{...}} fromNow={false} toNow={false} calendar={false} ago={false} unix={false} utc={false} local={false} unit={{...}} withTitle={false} decimal={false} titleFormat=\\"\\" interval={60000} filter={[Function: filter]} onChange={[Function: onChange]} />
  </p>
  <p>
    <strong>
      Degree or Certification: 
    </strong>
     
  </p>
  <p>
    <strong>
      Field of Study: 
    </strong>
     
  </p>
  <p>
    <strong>
      Description: 
    </strong>
     
  </p>
</div>"
`;
